save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/validation/ValidationUtils.kt

```diff
@@ -8,6 +8,7 @@ package com.saveourtool.save.validation
  * Default amount of characters allowed for names
  */
 const val NAMING_ALLOWED_LENGTH = 64
+const val NAMING_MAX_LENGTH = 22
```

Comment:
Why `allowed` is bigger than `max`?

Reply:
Because if something is impossible, but you really want it, then sometimes you can :)